### PR TITLE
Support improved Node Cellar simple blueprint

### PR DESCRIPTION
**Pull request by tliron**
In conjunction with this pull request:

https://github.com/cloudify-cosmo/cloudify-nodecellar-example/pull/91

### CODE REVIEW

**Comment by tliron** [comment]
Note that the CircleCI tests will obviously fail until the cloudify-nodecellar-example pull request is merged into its master.

File: cosmo_tester/test_suites/test_simple_manager_blueprint/nodecellar_singlehost_test.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ def get_public_ip(self, nodes_state):
 
     def get_inputs(self):
         return dict(self.access_credentials,
-                    **{'host_ip': self.private_ip_address})
+                    **{'nodejs_host_ip': self.private_ip_address,
+                       'mongod_host_ip': self.private_ip_address})
 
     @property
     def expected_nodes_count(self):
```
Comment (EarthmanT) [inline cosmo_tester/test_suites/test_simple_manager_blueprint/nodecellar_singlehost_test.py]: should now be 5

Comment (EarthmanT) [inline cosmo_tester/test_suites/test_simple_manager_blueprint/nodecellar_singlehost_test.py]: entrypoint_node_name should be nodejs_host

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**